src/main/webapp/task.html

```diff
@@ -7,16 +7,31 @@
         <script src="script.js"></script>
     </head>
     <body onload="loadTasks()">
-    <h1 class="task-title">TO-DO LIST</h1>
-    <form method="POST" action="/new-task">
-        <input type="text" name="title" />
-        <br/>
-        <button>Submit</button>
-    </form>
+        <div class="top-container">
+            <div class="dropdown">
+                <button class="dropbtn">MENU</button>
```

Comment:
Fyi, you can actually do capitalization via an allcaps property: https://www.w3schools.com/cssref/pr_text_text-transform.asp